pages/my-reports/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+		<!-- WPA -->
+		<link rel="manifest" href="../../manifest.json" />
+		<link
+			rel="apple-touch-icon"
+			href="../../assets/css/img/icons/icon-96x96.png"
+		/>
+		<meta name="apple-mobile-web-app-status-bar" content="#000000" />
+		<meta name="theme-color" content="#000000" />
+
+		<!-- leaflet CSS for maps -->
+		<!-- <link rel="stylesheet" href="../../modules/leaflet/leaflet.css"> -->
+		<!-- leaflet JS for maps -->
+		<!-- <script src="../../modules/leaflet/leaflet.js"></script> -->
+		<!-- Firebase JS -->
+		<!-- <script src="https://www.gstatic.com/firebasejs/ui/6.0.1/firebase-ui-auth.js"></script> -->
+		<!-- Firebase CSS -->
+		<!-- <link type="text/css" rel="stylesheet" href="https://www.gstatic.com/firebasejs/ui/6.0.1/firebase-ui-auth.css" /> -->
+		<!-- Bootstrap JS -->
+		<script src="../../modules/bootstrap/js/bootstrap.min.js"></script>
+		<!-- Bootstrap CSS -->
+		<link
+			rel="stylesheet"
+			href="../../modules/bootstrap/css/bootstrap.min.css"
+		/>
+		<!-- Variables -->
+		<link rel="stylesheet" href="../../assets/css/variables.css" />
```

Comment:
We no longer have to link _variables because is imported inside _global

Reply:
fixed in https://github.com/whamester/WMDD-4885-Project/pull/24/commits/38be8ad4f105b14b84a1b7621dc6974c2bcc4be3